packages/sdk/src/actor/actor.ts

```diff
@@ -458,40 +471,45 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * If omitted, a best-guess size is chosen based on the currently assigned mesh
 	 * (loading meshes are not considered). If no mesh is assigned, defaults to (1, 1, 1).
 	 * @param center The center of the collider, or default of the object if none is provided.
+	 * @param layer Controls what the assigned actors will collide with.
 	 */
 	public setCollider(
 		colliderType: ColliderType.Capsule,
 		isTrigger: boolean,
 		size?: Vector3Like,
-		center?: Vector3Like
+		center?: Vector3Like,
+		layer?: CollisionLayer,
```

Comment:
Collision layers was disabled here for a specific reason as we do not support changing the collision layer after the actor has been created.  If there is a need for this to meet some kind of functional reuirement, then please pull this out of this PR and submit a feature request PR with the change so that we can discuss it there.